### ISSUE TITLE
Connection Refused with the Percona Mysql when starting Mautic 

### ISSUE THREAD

**Issue by goodscrimshaw** [mautic/docker-mautic#132]
Could use some advice on how to trouble shoot. 

I used the exact step by step terminal commands titled "Running Basic Container".
![image](https://user-images.githubusercontent.com/56464509/67059936-fdc45880-f128-11e9-985a-380492dd6385.png)

This is what I am getting in the Mautic container log and nothing for local host in the browser. 

![image](https://user-images.githubusercontent.com/56464509/67059248-47f80a80-f126-11e9-8204-aaa101dc73ad.png)

I see #112 had a similar error using docker compose. Checking like @bohuspollak suggested the percona container IP returned the following: 
"IPAddress": "172.17.0.2"

This lead me to modify my terminal code similarly:
```
docker volume create mysql_data

docker run --name percona -d -p 3306:3306 -e MYSQL_ROOT_PASSWORD=mypassword -v mysql_data:/var/lib/mysql percona/percona-server:5.7 --character-set-server=utf8mb4 --collation-server=utf8mb4_general_ci

docker volume create mautic_data

docker run --name mautic -d  --restart=always  -e MAUTIC_DB_HOST=127.17.0.2  -e MAUTIC_DB_USER=root  -e MAUTIC_DB_PASSWORD=mypassword  -e MAUTIC_DB_NAME=mautic  -e MAUTIC_RUN_CRON_JOBS=true -e MAUTIC_TRUSTED_PROXIES=0.0.0.0/0 -p 8081:81 -v mautic_data:/var/www/html  mautic/mautic:latest
```
That produces the following log erros
![image](https://user-images.githubusercontent.com/56464509/67059724-2ef05900-f128-11e9-8bcb-bf8e03a93602.png)

I am pretty sure I am missing something basic here but I cant seem to figure it out. Is there a connection refusal going on due to a system settings somewhere?  I am running docker desktop 2.1.0.3. I am very new to docker and this sort of thing. Any advice would be greatly appreciated.

**Comment by goodscrimshaw** [comment]
I got it

**TLDR: It works if you modify the running mautic code listed on the [README](https://github.com/mautic/docker-mautic#running-basic-container) by following steps to find IP in #112 BUT ALSO you specify the DB port after the IP.**
 _Using the example results correct IP of 172.17.0.2 and DB port of 3306_
```
-e MAUTIC_DB_HOST=127.17.0.2:3306
```
Using the steps in #112 to find your IP, my thought was to just update Running Mautic code listed on [README](https://github.com/mautic/docker-mautic#running-basic-container)  with the updated IP like so the following:
```
$ docker run --name mautic -d \
    --restart=always \
    -e MAUTIC_DB_HOST=172.17.0.2 \
    -e MAUTIC_DB_USER=root \
    -e MAUTIC_DB_PASSWORD=mypassword \
    -e MAUTIC_DB_NAME=mautic \
    -e MAUTIC_RUN_CRON_JOBS=true \
    -e MAUTIC_TRUSTED_PROXIES=0.0.0.0/0 \
    -p 8080:80 \
    -v mautic_data:/var/www/html \
    mautic/mautic:latest
```

That does not work. Throws the same error. But putting the port after makes it run. 

```
$ docker run --name mautic -d \
    --restart=always \
    -e MAUTIC_DB_HOST=172.17.0.2:3306 \
    -e MAUTIC_DB_USER=root \
    -e MAUTIC_DB_PASSWORD=mypassword \
    -e MAUTIC_DB_NAME=mautic \
    -e MAUTIC_RUN_CRON_JOBS=true \
    -e MAUTIC_TRUSTED_PROXIES=0.0.0.0/0 \
    -p 8080:80 \
    -v mautic_data:/var/www/html \
    mautic/mautic:latest
```

**Comment by lquessenberry** [comment]
Worked solid for me!
